source/dem/force_chains_visualization.cc

```diff
@@ -455,21 +456,23 @@ ParticlesForceChains<dim, contact_model, rolling_friction_model>::
                                   ParticleParticleContactForceModel::
                                     hertz_mindlin_limit_overlap)
                     {
-                      this->calculate_hertz_mindlin_limit_overlap_contact(
-                        contact_info,
-                        tangential_relative_velocity,
-                        normal_relative_velocity_value,
-                        normal_unit_vector,
-                        normal_overlap,
-                        particle_one_properties,
-                        particle_two_properties,
-                        normal_force,
-                        tangential_force,
-                        particle_one_tangential_torque,
-                        particle_two_tangential_torque,
-                        rolling_resistance_torque);
+                      this
```

Comment:
Careful, you did not change the value at line 150 for the overlap

Reply:
I need to check for the force chains, you are right.